### ISSUE TITLE
[task] SC Design Updates

### ISSUE THREAD

**Issue by eileen-coforma** [department-of-veterans-affairs/va.gov-team#87098]
## Value Statement

**_As a_** Veteran

**_I want to_** submit the most up-to-date form

**_So that_** my form will get accepted with all the necessary data without extra processing time

---

## Background Context

20-0995 was updated in June 2024. We want to match the web portal to the new paper form. Paper form includes new sections that need new screens to capture.

## Outcome, Success Measure, KPI(S), and Tracking Link

- Add here

## Acceptance Criteria

- [ ]  Designs files are updated
- [ ]  Updates meet requirements
- [ ]  Designs reviewed by enablement

## Designs and Build Notes

- [FIGMA](https://www.figma.com/design/2LGebZcUuu5Iqh4QLPII6A/Supplemental-Claims-(VA-0995)?node-id=0-1)

### Enablement team (if needed)
@jstrothman 

## Out of scope

- Add here

## Open questions

- Add here

## Tasks

- [ ] Add new VHA Notifications questions
- [ ] Add new medical records questions
- [ ] Update content based on new form

## Definition of Ready

- [ ] Clear value description
- [ ] Testable acceptance criteria
- [ ] Accessibility added to acceptance criteria
- [ ] Approved designs attached
- [ ] Sample data provided where appropriate
- [ ] Estimated to fit within the sprint
- [ ] Dependencies and blockers linked

## Definition of Done

- [ ] Reviewed and approved by OCTO and VBA stakeholders

**Comment by eileen-coforma** [comment]
Looking to kick off CAIA conversation for flow + content review. Still collaborating with Jina Ryu and Katherine Fung on the VHA Notifications screen.

**Comment by eileen-coforma** [comment]
Pinged CAIA for review.

**Comment by eileen-coforma** [comment]
Meeting with CAIA OH on Tuesday to discuss new UX flow because of updated/new screens.

**Comment by Mottie** [comment]
Moving SME edits into Figma

**Comment by Mottie** [comment]
Updating new SC content from CAIA.

**Comment by eileen-coforma** [comment]
Added new screen for filtering users for the MST VHA notifications question.
- Still waiting on SME approval for MST VHA notifications
- CAIA approves new UX flow

Working on SC prototype.

**Comment by anniebtran** [comment]
Still waiting on SME, Eileen left message in CAIA ticket to see if we can move forward

**Comment by anniebtran** [comment]
CAIA has confirmed can move forward with updates, SME should respond to comments today; will follow up tomorrow if not

**Comment by anniebtran** [comment]
Eileen has updated designs based on content, SME will respond on Wed

**Comment by anniebtran** [comment]
SMEs got back to us, Eileen updated screen and will answer questions about prototype

**Comment by anniebtran** [comment]
Designs need some updates bc CAIA confirmed some updates

**Comment by eileen-coforma** [comment]
I'm pretty sure I completed these. I'll open another ticket if there are changes beyond the initial suggestions.
